java/ql/src/semmle/code/java/dataflow/internal/DataFlowImplCommon.qll

```diff
@@ -213,6 +213,16 @@ private module ImplCommon {
     argumentValueFlowsThrough(node1, node2, _)
   }
 
+  private predicate parameterValueFlowNoCtx(ParameterNode p, Node node) {
```

Comment:
`parameterValueFlowNoCtx` still uses flow through callables as that's included in `localValueStep`.  The difference between `parameterValueFlowCand` and `parameterValueFlowNoCtx` is that the latter is a slight precision improvement due to call context being used in the nested flow through callables.  I wanted to remove the direct dependency on `parameterValueFlowCand` so it can be private in the nested module that's introduced in the next commit, and with that change it makes more sense to define `parameterValueFlowNoCtx` like this based on the resulting `argumentValueFlowsThrough(node1, node2, _)` - a side-effect is then a slight precision improvement for the detection of getters and setters.

Reply:
Would be nice with a test case that covers this.